### DISCUSSION TITLE
Seamless Connection (3rd party device compatibility)

### DISCUSSION THREAD

**Discussion by sccccchallz** [timschneeb/GalaxyBudsClient#227] [Q&A] [answered]
How could I make the seamless connection work properly? I did it when the first time I use this program just like the airpods does on iOS and iPadOS. But after that, I have not done that 'seamless' cause I have to open the Wearable than it will connect to my phone. Therefore, I do not know what I did wrong on any point though.

By the way, I really like the pop-up, thank you for bringing me that feature on PC and I could get better connection process experience on that.

**Comment by ArthurWolfhound** [comment]
(Contributor here) I may be wrong, but seamless connection might only work on Samsung Devices...


BTW, there's a new discussion feature so anyone can talk there without opening issues.

P.S.: I will convert that into discussion shortly.

**Comment by sccccchallz** [comment]
It just seems to be with that feature but not able to work well all the time which means that it is not connected without retry several time on the PC. But the PC, while playing new media, is trying to connect the Buds and it is failed most of time.
And I just made it again.
Idk, probably the issue is on the phone

**Comment by ArthurWolfhound** [comment, marked as answer]
Just checked. Seamless connection is not available for 3rd party devices - and setting toggle is present for the sake of ability to toggle that setting.

More info here - [xda-developers](https://www.xda-developers.com/samsung-galaxy-buds-plus-now-easily-switch-between-devices-seamlessly/)

**Reply by sccccchallz** [reply]
Thanks
